visualization/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
     <img width="300" alt="color-metric-diagram" src="https://github.com/MaibornWolff/codecharta/assets/149768491/0b0e4e6f-e0c2-49b1-b9e8-2d69002338f0">
     </br>
 -   Persist uploaded files and applied configuration accross page-reloads [#3486](https://github.com/MaibornWolff/codecharta/pull/3486)
+-   Auto detect direction of metrics [#3518](https://github.com/MaibornWolff/codecharta/pull/3518)
```

Comment:
Could explain a bit more what this means, e.g. "Auto detect direction of metrics, meaning metrics where lower is better are now displayed differently than metrics where higher is better"